code-challenges.js

```diff
@@ -6,18 +6,49 @@ var secretCodeWord1 = "lackadaisical"
 // Expected output: "l4ck4d41s1c4l"
 var secretCodeWord2 = "gobbledygook"
 // Expected output: "g0bbl3dyg00k"
+// turn string into an array
+//iterate through array using .map to equate the values to the expected numbers
+//return the array back to a string using the .join method
+const codedMessage = (string) => {
+  let stringArr = string.split("")
+  let newArr = stringArr.map(value => {
+    if(value == "a"){
+       return value = 4
+    }
+    else if(value == "e"){
+      return value = 3
+    }
+    else if(value == "i"){
+      return value = 1
+    }
+    else if(value == "o"){
+      return value = 0
+    }
+  })
+  return newArr.join("")
+}
 
-
-
-
+console.log(codedMessage(secretCodeWord1))
+console.log(codedMessage(secretCodeWord2))
 
 // --------------------2) Create a function that takes in an array and returns all the words that contain the letter a.
 
 var arrayOfWords = ["Apple", "Banana", "Plum", "Cherry", "Kiwi", "Peach"]
 // Expected output: "Apple" "Banana" "Peach"
 
+const onlyA = (array) => {
+  smallArr = []
+    for(var i = 0; i < array.length; i++){
+      smallArr.push(array[i].toLowerCase())
+    }
+    newArray = smallArr.filter(value => {
+      return value = value.includes("a")
```

Comment:
here, you could do `value.toLowerCase().includes("a")` to eliminate the need for the loop above